SETUP.md

```diff
@@ -0,0 +1,184 @@
+# Setting up your Cook dev environment
+
+This document tells you how to set up a Cook dev environment from
+scratch on a Macintosh computer.
```

Comment:
Thanks! 

The notes are really just telling you to install a bunch of software on your system.  The procedure would be very similar on Linux, just with "apt-get" or "yum" instead of "port".  I'll add a note to that effect.